stock/templates/registered/sigtracker.html

```diff
@@ -0,0 +1,45 @@
+{% extends "public/base.html" %}
+{% load staticfiles %}
+
+{% block title %}Alliance Auth{% endblock %}
+
+{% block page_title %}Signature Tracker{% endblock page_title %}
+{% block extra_css %}{% endblock extra_css %}
+
+{% block content %}
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title></title>
+    </head>
+    <body  bgcolor="silver">
+        
+
+        <div align="center"> 
+            <form action="form1.php" method="post">
```

Comment:
are you embedding a php form in this html page? What purpose does it serve? Why not use a django form?

What does this page even do?